src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -66,9 +66,9 @@
       <property name="geometry">
        <rect>
         <x>0</x>
-        <y>0</y>
+        <y>-435</y>
```

Comment:
This change is strange, but I guess it's right. Do you know why it could have happened?

Reply:
I don't know why this is happening I edited everything through qt designer. I'm guessing this is correct since the menu shows up correctly and there's no need to change this right now.